sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_content_safety/_hate_unfairness.py

```diff
@@ -71,6 +72,7 @@ def __init__(
             eval_metric=EvaluationMetrics.HATE_FAIRNESS,
             azure_ai_project=azure_ai_project,
             credential=credential,
+            conversation_aggregation_type=AggregationType.MAX,
```

Comment:
The behavior was changed from mean to max. Should we treat it a bug fix or a breaking change?

Reply:
This is intentional, and the main reason I started this PR in the first place. CS evals aggregating multi-turn results via a mean instead of a max is considered a bug, so this is fixing that. I'll move the CL comment of that into the bugfix section.